app/locales/ru.json

```diff
@@ -1,193 +1,193 @@
-    "notifications": "Break notifications:",
-    "notificationsInfo": "System notification shown before breaks to let you prepare for it.",
+    "title": "Настройки участников",
+    "notifications": "Уведомления о перерывах:",
+    "notificationsInfo": "Системные уведомления, показвающиеся перед перерывами, чтоб дать время приготовиться.",
     "beforeLongBreak": "Long Break:",
     "beforeMiniBreak": "Mini Break:",
-    "newVersion": "New version:",
-    "newVersionNotification": "Notify when new version is available",
-    "customIdeas": "Custom ideas:",
-    "customIdeasInfo": "Allows you to use custom break ideas by editing configuration file. (For advanced users.)",
-    "useIdeasFromSettings": "Use break ideas from configuration file",
-    "breakPostpone": "Postponement of breaks:",
-    "breakPostponeInfo": "If enabled in main Preferences, you can postpone breaks. Following settings allow you to specify when, for how long and how many times you can postpone a break.",
-    "miniBreaks": "Mini Breaks:",
-    "postponableFor": "Postponable for:",
-    "postponeFor": "Postpone for:",
-    "maxPostpones": "Max of postpones:",
-    "longBreaks": "Long Breaks:",
-    "pauseUntilMoring": "Pause until morning:",
-    "pauseUntil": "Pause until:",
-    "pauseUntilMorningInfo": "Specify till what hour 'Pause until morning' should pause.",
-    "welcomeWindow": "Welcome window:",
-    "showWelcomeWindow": "Show Welcome window on next start",
-    "sounds": "Sounds:",
-    "volume": "Volume:",
-    "miniBreakStartSound": "Play end-of-break sound also at the start of Mini break",
-    "longBreakStartSound": "Play end-of-break sound also at the start of Long break",
-    "naturalBreaks": "Natural breaks:",
-    "naturalBreaksInfo": "If monitoring of idle time is enabled in main Preferences, you can specify after what time the countdown should be paused.",
-    "pauseAfter": "Pause after:",
-    "appearance": "Appearance:",
-    "light": "Light",
-    "dark": "Dark",
-    "system": "System"
+    "newVersion": "Новая версия:",
+    "newVersionNotification": "Уведомлять о наличии обновлений",
+    "customIdeas": "Свои идеи для упражнений:",
+    "customIdeasInfo": "Позволяет настроить свои собственные идеи для перерывов в конфигурационном файле (для продвинутых пользователей).",
+    "useIdeasFromSettings": "Использовать идеи упражнений из конфигурационного файла",
+    "breakPostpone": "Возможность откладывать перерывы:",
+    "breakPostponeInfo": "Если этот пункт включен в основных настройках, вы можете откладывать перерывы. Последующие настройки позволяют указать, когда, на какой промежуток временни и сколько раз можно откладывать перерывы.",
+    "miniBreaks": "Мини-перерывы:",
+    "postponableFor": "Можно отложить в течение:",
+    "postponeFor": "Откладывать на:",
+    "maxPostpones": "Сколько раз можно откладывать:",
+    "longBreaks": "Длинные перерывы:",
+    "pauseUntilMoring": "Приостановить до утра:",
```

Comment:
Not sure why there's a colon here. 
Are the following 3 entries all part of one form field?